packages/wonder-blocks-form/src/components/__tests__/text-field.test.tsx

```diff
@@ -740,4 +746,901 @@ describe("TextField", () => {
         const input = await screen.findByRole("textbox");
         expect(input).toHaveAttribute("aria-invalid", "false");
     });
+
+    describe("Validation", () => {
+        describe("validate prop", () => {
+            it("should be in an error state if the initial value is not empty and not valid", async () => {
+                // Arrange
+                render(
```

Comment:
**suggestion:** This would go in `Act` as it is the part of the test that triggers the assertion.

Reply:
I cleaned up the Arrange/Act comments! I see now the initial render can be both the Arrange and Act part, thanks for letting me know!